src/main/java/com/passbook/sparkeighteen/peristence/POJO/ProfileRequest.java

```diff
@@ -35,18 +35,18 @@ public class ProfileRequest {
     private String gender;
 
     //    @DateTimeFormat(pattern="dd/MM/yyyy")
-    private LocalDate date_of_birth;
+    private LocalDate dateOfBirth;
 
     @NotEmpty(message = "address is mandatory")
     private String address;
 
     @NotBlank
     @NotEmpty(message = "pan number is mandatory")
-    private String pan_number;
+    private String panNumber;
 
     @NotBlank
     @NotEmpty(message = "aadhar number is mandatory")
-    private BigInteger aadhar_number;
+    private BigInteger aadharNumber;
```

Comment:
Can you let me know the limit of BigInteger? (i.e max value supported by it )

Reply:
BigInteger takes 8 bytes i.e. 64bits. 
Signed range :  -9223372036854775808 to 9223372036854775807
unsigned Range takes positive value : 0 to 18446744073709551615